### PR TITLE
feat: Add May Eleven Links

### LINKED ISSUE #442: [Intermediate]: Write a Blog Post On Hiero Links For The Week of May 11th
### 🧑‍💻 Intermediate Issue

Welcome! This is an **[Intermediate Issue](https://github.com/issues?q=is%3Aopen%20is%3Aissue%20org%3Ahiero-ledger%20archived%3Afalse%20no%3Aassignee%20label%3A%22intermediate%22%20repo%3Ahiero-ledger%2Fhiero-website)** touching core website architecture.

🏁 **When this issue is complete, you will have:**

✅ Researched across multiple files and modules
✅ Owned implementation decisions
✅ Written thorough, meaningful tests
✅ Delivered a clean, review-ready pull request

### 🐞 Problem Description

Currently there is little information on what has been happening in the Hiero Organization on the Hiero Website.
This is a problem as new starters may want to become familiar with what has changed recently and what is new in Hiero.

We propose to add a blog post containing links to things that have occurred during the week of May 4th.

### 💡 Expected Solution

The blog includes:

* Links to relevant changes within the Hiero Organization
* Names of the individuals who created the changes/updates
* A way to contact the Hiero Team if they would like to be featured.

### 🔍 Background Research

**Blog & Content Workflow:**
- [Writing Blog Posts](https://github.com/hiero-ledger/hiero-website/blob/main/docs/blogs.md) — covers the current posting flow, front matter format, and content conventions

**Project Conventions:**
- Components live in `src/components/` — follow existing patterns for props, naming, and file structure
- Pages are under `src/app/` using the Next.js App Router
- Content (blog posts, etc.) lives in `content/`
- Styling uses [Tailwind CSS](https://tailwindcss.com/docs)

### 🛠️ Implementation Notes

The blog will need to be written in markdown and we have a guide here [Blog.md](https://github.com/aceppaluni/hiero-website/blob/main/docs/blogs.md)

For information on recent changes reach out to maintainers and listen to Hiero Meeting Recordings.

You can view meeting recordings here:
https://zoom-lfx.platform.linuxfoundation.org/meetings/hiero?view=week

But you can also look at merged pull requests throughout repositories in hiero:
e.g.

https://github.com/hiero-ledger/hiero-sdk-python/pulls?q=is%3Apr+is%3Aclosed

The final style should be similar to
https://hiero.org/blog/hiero-links-for-the-week-of-february-2nd/

### 🔬 Technical Domains

      

- [ ] **Intermediate to Advanced Web Development** (React components, hooks, state management, Next.js routing)
- [ ] **Component Architecture** (reusable components, props design, composition patterns)
- [ ] **Styling & Responsiveness** ([Tailwind CSS](https://tailwindcss.com/docs), responsive design, accessibility)
- [ ] **Performance** (image optimization, lazy loading, Core Web Vitals)
- [ ] **Testing** (unit, integration, snapshot, accessibility testing)

### 🧠 Intermediate Contributors — Prerequisites & Expectations

> [!CAUTION]
> **Intermediate issues are high-risk. We expect more than just a 'working solution' and will recommend beginner issues if the PR does not meet these standards.**

### 🏁 Concrete Prerequisites
- **Web Development Skills:** Intermediate or advanced React/TypeScript/Next.js skills.
- **Expertise:** Strong understanding of files related to this issue (research before claiming!).
- **Proven History:** Successfully completed **≥ 5 beginner issues** in this repo.

> [!NOTE]
> **CI/CD Exception:** For issues focused on **GitHub Actions / Workflows**, the repo-specific thresholds above may be waived if the contributor demonstrates intermediate-level proficiency in CI/CD.

If this feels like too big a step, that is completely fine — try a [Beginner Issue](https://github.com/issues?q=is%3Aopen%20is%3Aissue%20org%3Ahiero-ledger%20archived%3Afalse%20no%3Aassignee%20label%3A%22beginner%22%20repo%3Ahiero-ledger%2Fhiero-website) first or a different **[Intermediate Issue](https://github.com/issues?q=is%3Aopen%20is%3Aissue%20org%3Ahiero-ledger%20archived%3Afalse%20no%3Aassignee%20label%3A%22intermediate%22%20repo%3Ahiero-ledger%2Fhiero-website)**. You can always come back when you are ready.

### ⚠️ AI Usage Policy

- Using AI to generate code for Intermediate issues is **strictly discouraged**
- Using AI as the main source of research is **strictly discouraged**, refer to React, Next.js, and TypeScript documentation.

### ⏱️ Timeline & Workflow
- **Typical time:** ~2 weeks / ~25 hours.
- 🔴 Completing an intermediate issue in 1–3 days is a **red flag**.

> [!TIP]
> **Suggested:** share your proposed implementation approach as a comment before writing code to get early feedback and avoid wasted effort.

### 🧪 Testing Requirements

How you test depends on the type of change.

> [!IMPORTANT]
> At the intermediate level, **testing is a major component**.
> Each component or function you implement should be verified. Tests should cover happy paths,
> edge cases, and error handling.

**Component or source code changes (e.g. in `src/`):**
- Write unit tests covering happy path, edge cases, and error handling
- Run tests locally: `pnpm test`
- Verify the dev server renders correctly: `pnpm dev`
- Check consistency with similar components already in the codebase — use the same patterns

**GitHub Actions / workflow changes (e.g. in `.github/`):** [Guide](https://github.com/hiero-ledger/sdk-collaboration-hub/blob/main/guides/issue-progression/for-developers/github-action-workflows.md)
- Test by merging to your fork's `main` and simulating the scenario
- Create test issues or PRs as evidence and link them in your PR
- Include screenshots of workflow runs where applicable

**Content, styling, or layout changes:**
- Run the dev server and confirm visual output matches expected behavior
- Compare your changes with similar existing pages for consistency
- Check responsive behavior and accessibility

### 🛡️ Quality & Review Standards

Intermediate PRs must be **"working, maintainable, and aligned with project architecture."**

  1. **Working:** The implementation must solve the problem and meet the acceptance criteria.
  2. **Maintainable:** Code should be clear and concise enough for others to understand and debug without your assistance.
  3. **Architectural Alignment:** The solution should fit well with existing component patterns and project conventions.
  4. **Backward Compatibility:** Existing pages and components must not break. 
  5. **Comprehensive Testing:** Must include unit tests covering all new logic paths, edge cases, and failure modes. AI-generated tests based on AI-generated code are grounds for immediate rejection.

**⚠️ Breaking changes**
- Before changing any component API, prop interface, or public-facing layout — stop and check
- If a breaking change is unavoidable: get explicit maintainer approval **before** implementing
- All existing tests should pass as-is.

**🤖 AI notes**
- AI often has out-dated real-world knowledge: it will not be able to identify current packages.
- AI has gaps or incorrect understanding of repo-specific patterns, logic, and component structures.

### ✅ PR Quality Checklist

Before opening your PR, confirm:

- [ ] I have spent the majority of my time researching the problem and solution space extensively, including reviewing relevant code, documentation, and external resources.
- [ ] My implementation works but is also of high quality - including maintainability, readability, and architectural fit.
- [ ] I have checked for breaking changes - no existing pages or components regress.
- [ ] The design fits with the current website's architectural approaches.
- [ ] Every line of code is personally understood and explainable.

Before requesting a review, confirm:
- [ ] I have reviewed the diff line by line.
- [ ] My implementation fully addresses the problem described above.
- [ ] I did not modify files unrelated to this issue
- [ ] Clean git history — no rebase artifacts, merge commits, or unrelated files
- [ ] My commits are signed: `git commit -S -s -m "chore: description"` — [Signing guide](https://github.com/hiero-ledger/sdk-collaboration-hub/blob/main/guides/issue-progression/for-developers/signing.md)
- [ ] I have run the following commands locally and **all pass** (CI will automatically fail your PR if any of these fail):
  - `pnpm format:check` — checks code formatting
  - `pnpm lint` — runs the linter (**the CI lint check will block your PR if this fails**)
  - `pnpm test` — runs the test suite
  - `pnpm build` — ensures the project builds successfully
  - [Testing guide](https://github.com/hiero-ledger/hiero-website/blob/main/docs/06-testing-and-quality-checks.md)
- [ ] I have included appropriate tests and all CI checks pass.
- [ ] Double and triple check — intermediate PRs are time-consuming to review

### 📚 Resources & Support

**🆘 Stuck?**
> [!TIP]
> **Comment on this issue:** and describe what you have tried. A maintainer will respond.

**References:**
- [Next.js Documentation](https://nextjs.org/docs)
- [React Documentation](https://react.dev)
- [Tailwind CSS Documentation](https://tailwindcss.com/docs)
- [Vitest Documentation](https://vitest.dev)
- [Writing Blog Posts](https://github.com/hiero-ledger/hiero-website/blob/main/docs/blogs.md)
- [Community Calls](https://zoom-lfx.platform.linuxfoundation.org/meetings/hiero?view=week)
- [Discord](https://github.com/hiero-ledger/sdk-collaboration-hub/blob/main/guides/issue-progression/for-developers/discord.md)

### PR DESCRIPTION
**Pull request by aceppaluni**
# Description

This PR adds the links to things that happened within Hiero during the week of May 11th

## Changes Made
- [x] Added... A markdown file and corresponding image banner.
- [ ] Modified...
- [ ] Fixed...

## Related Issues

Fixes: #442 

## Checklist
- [ ] Tests added/updated
- [ ] Documentation updated
- [ ] Linting passes
- [x] Branch up-to-date with main

## Release Notes

* **Documentation**
  * New weekly blog post showcasing featured community contributions and highlighted pull requests.

[Review Change Stack](https://app.coderabbit.ai/change-stack/hiero-ledger/hiero-website/pull/443)

### CODE REVIEW

File: content/posts/hiero_links_of_may_eleven.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+**Angelina Ceppaluni** implemented [HIP 1261:Simple Fees](https://github.com/hiero-ledger/hiero-sdk-python/pull/2231) to support fee estimation in the Python SDK.
+
+**Achintya Sinha** corrected a windows [debug build failure](https://github.com/hiero-ledger/hiero-sdk-cpp/pull/1623) in FeeEstimateQuery.cc by correcting the include order to prevent MSVC macro collisions in the C++ SDK. 
+
+**Xin Li** implemented [TSS system transaction types](https://github.com/hiero-ledger/hiero-mirror-node/pull/13525) to REST to ensure TSS system transaction types are properly recognized in the REST transactions endpoint instead of appearing as unknown transaction types in Hiero Mirror Node.
+
+**Samrajniii** added a [broken link checker](https://github.com/hiero-ledger/hiero-sdk-java/pull/2729) script to check for broken links in the Java SDK.
+---
```
Comment (danielmarv) [inline content/posts/hiero_links_of_may_eleven.md]:
```suggestion

---
```
it is making the paragraph above it bold

**Review by danielmarv — CHANGES_REQUESTED** [review]

**Review by danielmarv — APPROVED** [review]
